common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -446,6 +447,29 @@ def commits_details(base:, head:)
           end
         end
 
+        context "and capitalizes the message but not the prefix" do
+          before do
+            stub_request(:get, watched_repo_url + "/commits?per_page=100").
+              to_return(
+                status: 200,
+                body: fixture("github", "commits_angular_sentence_case.json"),
+                headers: json_header
+              )
+          end
+
+          it do
+            is_expected.to eq("chore(deps): Bump business from 1.4.0 to 1.5.0")
+          end
+
+          context "and with commit messages explicitly configured" do
+            let(:commit_message_options) { super().update(prefix: "chore(dependencies)") }
```

Comment:
I appreciate your use of `super()` here because it signals that I need to look up the chain. I'm a fan of RSpec, but one of my least favorite parts of it is that it can lead to an explosion of mystery guests. Because of this, I wonder if we could use `#merge`, which I think of as being a `Hash`-specific method. I think it gives 🤏 more context to the variable, and takes away a little bit of ambiguity _for me_.

```suggestion
            let(:commit_message_options) { super().merge(prefix: "chore(dependencies)") }
```

If this goes against your instincts, please feel free to ignore 😄

Reply:
Using `#merge` feels better to me too, not sure why I used `update`!